src/der.rs

```diff
@@ -277,4 +317,11 @@ mod tests {
             });
         }
     }
+
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha256_key_encoded_ber,
+                               "rsa-pkcs1-sha256-key-encoded-ber.pem" );
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha1_bad_key_der_length,
+                               "rsa-pkcs1-sha1-bad-key-der-length.pem");
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha1_bad_key_der_null,
+                               "rsa-pkcs1-sha1-bad-key-der-null.pem");
```

Comment:
Looks like `travis.sh` needs to copy over the chromium `*.pem` files like it copies over the `txt` test files. However, the current setup also isn't running integration tests (like the new spki ones) since it only copies over `target/(release|debug)/ring-*` and the test file for spki is `target/(release|debug)/signature_spki_tests-*`.